transport/processing/processing.py

```diff
@@ -133,9 +132,9 @@ def _uploader(uploader_cfg: dict):
 
 
 def processing_pipeline(
-    processing_cfg: dict,
-    shared_processors: dict={},
-    shared_uploaders: dict={},
+        processing_cfg: dict,
+        shared_processors: dict = {},
```

Comment:
general remark: it can lead to unexpected side-effects if using mutable values as default values.

consider e.g.:

```
def foo(x,a={}):
 for k,v in x.items():
  a[k] = v
 print(a)
>>> foo({1:1})
{1: 1}
>>> foo({2:1})
{1: 1, 2: 1}
>>> foo({3:1})
{1: 1, 2: 1, 3: 1}
```

Reply:
that being said: in this particular case, it will pbly not be an issue (+ we do have several similar constructs in cc-utils, currently - I only learnt about this not foo far back, when reading through @dataclasses..)

Reply:
hmm that's a really good point in general I was not aware of.

It is not an issue in this case because we set a default value before, when we look in the processing configuration. Since they are not useful, I will remove them.